profile_scrapper_json_schema.json

```diff
@@ -0,0 +1,141 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://example.com/example.json",
+    "title": "Json schema for profile scraper",
+    "description": "Json schema for profile scraper",
+    "type": "object",
+    "properties": {
+        "url": {
+            "type": "string",
+            "description": "URL of the person's profile"
+        },
+        "name": {
+            "type": "string",
+            "description": "Name of the person"
+        },
+        "location": {
+            "type": "string",
+            "description": "Location of the person"
+        },
+        "experience": {
+            "type": "array",
+            "description": "Experience of the person",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "title": {
+                        "type": "string",
+                        "description": "Title of the job"
+                    },
```

Comment:
This is good but actually, it won't be directly usable in the code. I'd say use Python types instead. Learn about `typing` std lib in Python and maybe `pydantic` or `dataclasses`

Reply:
ok... will do that